src/Emitter.ts

```diff
@@ -239,6 +667,20 @@ export class Emitter
 
     private _parseTreeNode(node: IDocletTreeNode, parent?: IDocletTreeNode): ts.Node | null
     {
+        if ((this.options.generationStrategy === 'exported') && (! node.isExported))
+        {
+            debug(`Emitter._parseTreeNode(${docletDebugInfo(node.doclet)}): skipping doclet, not exported`);
+            return null;
+        }
+        if ((! node.exportName) // Check the `.exportName` flag before calling `_ignoreDoclet()` in order to avoid false debug lines.
```

Comment:
```suggestion
        if (!node.exportName // Check the `.exportName` flag before calling `_ignoreDoclet()` in order to avoid false debug lines.
```

Reply:
Taken into account.